stories/DayPickerRangeController.js

```diff
@@ -68,6 +68,49 @@ const TestCustomInfoPanel = () => (
   </div>
 );
 
+function renderKeyboardShortcutsButton(buttonProps) {
+  const { ref, onClick, ariaLabel } = buttonProps || {};
```

Comment:
Why would buttonProps need to be optional here?

If it is, it should be defaulted in the signature, no?

Reply:
@ljharb I think when I wrote this I intended to make buttonProps optional to avoid future implementations breaking, and so I used the defaulting pattern I saw in monorail. But if it's cool with the `react-dates` squad I can just make it required

Reply:
Seems better to require everything until it needs to be optional.

Reply:
Cool, I made it required